Source/SelfService/Web/documentation/container.tsx

```diff
@@ -81,6 +82,15 @@ export const DocumentationContainerScreen: React.FunctionComponent<Props> = (pro
                                     Verify access to kubernetes
                                 </a>
                             </li>
+                            <li>
+                                <a href="#" onClick={(event) => {
+                                    event.preventDefault();
+                                    const href = `/documentation/application/${applicationId}/${environment}/ci-cd/-azure-pipelines`;
```

Comment:
```suggestion
                                    const href = `/documentation/application/${applicationId}/${environment}/ci-cd/azure-pipelines`;
```

Was the `-` prefix intentional? Looks a bit odd to me.

Reply:
Fixed, good spot.